Xamarin.Forms.Platform.UAP/ImageRenderer.cs

```diff
@@ -10,13 +10,26 @@ public class ImageRenderer : ViewRenderer<Image, Windows.UI.Xaml.Controls.Image>
 	{
 		bool _measured;
 		bool _disposed;
-
+		Window _window;
 		public ImageRenderer() : base()
 		{
 			ImageElementManager.Init(this);
+			_window = Window.Current;
+			_window.Activated += Window_Activated;
```

Comment:
Do we need to unsubscribe this somewhere?

Reply:
Yes will do